src/main/java/hu/oe/nik/szfmv/automatedcar/virtualfunctionbus/packets/PowertrainPacket.java

```diff
@@ -0,0 +1,35 @@
+package hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.packets;
+
+public class PowertrainPacket implements ReadOnlyPowertrainPacket {
+
+    private int rpm;
+    private int speed;
+
+    public PowertrainPacket() {
+        this.rpm = 0;
+        this.speed = 0;
+    }
+
+    public PowertrainPacket(int rpm, int speed) {
+        this.rpm = rpm;
+        this.speed = speed;
+    }
+
+    @Override
+    public int getSpeed() {
+        return 0;
```

Comment:
Nyugodtan átadhatod már most a változó értékét, ha még igazából valós érték nem is lesz benn.
Legalább később már nem kell hozzányúlni :)

Reply:
Oks